docs/_includes/footer.html

```diff
@@ -6,8 +6,9 @@
     <ul class="bs-docs-footer-links">
       <li><a href="{{ site.repo }}">GitHub</a></li>
       <li><a href="https://twitter.com/getbootstrap">Twitter</a></li>
+      <li><a href="{{ site.baseurl }}/docs/{{ site.docs_version }}/getting-started/#examples">Examples</a></li>
       <li><a href="../getting-started/#examples">Examples</a></li>
```

Comment:
You need to delete the unnecessary line (duplicate "Examples").

Reply:
Done proposed changes